bin/varnishd/cache/cache_fetch.c

```diff
@@ -720,6 +720,7 @@ vbf_stp_error(struct worker *wrk, struct busyobj *bo)
 		bo->fetch_objcore->ttl = 1;
 		bo->fetch_objcore->grace = 5;
 		bo->fetch_objcore->keep = 5;
+		bo->stale_oc = NULL;
```

Comment:
I believe this leaks the stale_oc reference that is held at this point.

Reply:
yes, that was a stupid oversight from my side, thank you @mbgrydeland